src/imagealternatetext/ui/textareaview.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module image/imagealternatetext/ui/textareaview
+ */
+
+import View from '@ckeditor/ckeditor5-ui/src/view';
+import Template from '@ckeditor/ckeditor5-ui/src/template';
+import '../../../theme/textareaview/theme.scss';
+
+/**
+ * The TextAreaView class.
+ *
+ * @extends module:ui/view~View
+ */
+export default class TextAreaView extends View {
```

Comment:
BTW, it'd rather need to end up in the ckeditor5-ui. But, as I commented already, an input would make more sense to me.